osscameroon/camerdevs
# feat: add default jobtitles

This pull request set default values for the `GET /jobtitles` endpoint

Steps to verify:
- move to the `backend` folder with `cd ./backend`
- run `make start-api` to start the api
- then run the command `curl -X GET localhost:7000/jobtitles | jq` you should see something like this

```
[
  "2D Animator",
  "3D Animator",
  "3D Director",
   ...
  "Writer",
  "Yield Management Consultant",
  "ZEND Developer"  
]
```
